robert_gazebo/models/APPS_letters/model.sdf

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0'?>
+<sdf version='1.5'>
+    <model name="APPS_letters">
+        <pose>0 0 0 0 0 0</pose>
+        <static>true</static>
+        <link name='model_body'>
+            <visual name="visual">
+                <geometry>
+                    <mesh>
+                        <uri>model://robert_gazebo/meshes/APPS_letters_visual.dae</uri>
+                        <scale>1 1 1</scale>
+                    </mesh>
+                </geometry>
```

Comment:
please add collision and inertia

Reply:
This model is static and located on the wall, so robert won't have interaction with it.

Reply:
it is for now. But in future we might have some arm on top of Robert etc. If we are already spending time on models it good to build them with all required attributes :)